src/unstract/sdk/llm.py

```diff
@@ -68,15 +68,41 @@ def _initialise(self):
     def complete(
         self,
         prompt: str,
-        retries: int = 3,
+        process_text: Optional[Callable[[str], str]] = None,
         **kwargs: Any,
     ) -> Optional[dict[str, Any]]:
+        """Generates a completion response for the given prompt.
+
+        Args:
+            prompt (str): The input text prompt for generating the completion.
+            process_text (Optional[Callable[[str], str]], optional): A callable that
```

Comment:
NIT: Try to also document what the arguments to the `process_text` can be. In the future if needed, we can allow the user to pass args for that function as well

Reply:
Will take care of this later because it will be a internal use case for now.